CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### Fixed
 - Fixed issue which caused watch app schemes to be generated incorrectly, preventing these apps from launching. [#798](https://github.com/yonaskolb/XcodeGen/pull/798) @daltonclaybrook
+- Added build presets for the target type `framework.static`: due to the lack of them, archiving a static framework was resulting in an invalid archive. See https://github.com/yonaskolb/XcodeGen/pull/358 and https://stackoverflow.com/a/26563115/3203441 for further references. [#819](https://github.com/yonaskolb/XcodeGen/pull/819) @acecilia
```

Comment:
The context here is great! We can leave it in this PR though

Reply:
Context removed from `CHANGELOG.md`👌